### PR TITLE
facerecognition - use curl instead of wget

### LINKED ISSUE #2000: Face Recognition install Script Error.
the script fail to add the key automatically.

+ echo 'deb https://repo.delellis.com.ar focal focal'
+ sudo apt-key add -
+ wget -qO - https://repo.delellis.com.ar/repo.gpg.key
gpg: no valid OpenPGP data found.

### PR DESCRIPTION
**Pull request by szaimen**
Fix #2000
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: apps/face-recognition.sh [hunk 1]
```diff
@@ -90,7 +90,13 @@ fi
 # Install requirements
 # https://github.com/matiasdelellis/facerecognition/wiki/Installation#ubuntu-focal
 echo "deb https://repo.delellis.com.ar focal focal" > /etc/apt/sources.list.d/20-pdlib.list
-wget -qO - https://repo.delellis.com.ar/repo.gpg.key | sudo apt-key add -
+if ! curl -fsSL https://repo.delellis.com.ar/repo.gpg.key | sudo apt-key add -
```
Comment (enoch85) [inline apps/face-recognition.sh:93]: I was thinking the exact same thing actually, again. :D

But, I think maybe we should use the `curl_to_dir` function instead and improve that function for all the scripts with something like this.

Comment (szaimen) [reply apps/face-recognition.sh:93]: hm... can we do that in a follow up? 
It is honestly a bit out of scope for this PR and should better be done in a different PR, imo.

**Comment by szaimen** [comment]
This is ready 👍

**Comment by enoch85** [comment]
This was what I was thinking, or similar: https://github.com/nextcloud/vm/pull/2006

**Comment by szaimen** [comment]
Will be handled in #2006

**Pull request closed without merging**